lib/bing_ads_ruby_sdk/http_client.rb

```diff
@@ -26,19 +25,24 @@ class HttpClient
     class << self
       def post(request)
         uri = URI(request.url)
-        conn = connection(request.url)
```

Comment:
This was changed because the REST API requires that we request different URLs for each entity, unlike the SOAP API, in which we only call a handful of URLs, but change the SOAP Header to indicate which entity we wish to affect